caesars-palace/blocks/footer/footer.js

```diff
@@ -9,72 +15,102 @@ export default async function decorate(block) {
   const cfg = readBlockConfig(block);
   block.textContent = '';
 
+  let resp;
+
+  // add rewards
```

Comment:
conditionally embed content from rewards doc unless the `rewards` metadata has been set to `false`.